### DISCUSSION TITLE
How to simplify expression sqrt(9 - 4*sqrt(5)) ?

### DISCUSSION THREAD

**Discussion by satels** [sympy/sympy#23496] [Q&A] [answered]
Must be simplified to sqrt(9 - 4*sqrt(5))=sqrt(5) - 2

**Comment by oscarbenjamin** [comment, marked as answer]
You can use the `sqrtdenest` function for handling nested square roots (when possible):
```python
In [31]: e = sqrt(9 - 4*sqrt(5))

In [32]: e
Out[32]: 
  __________
╲╱ 9 - 4⋅√5 

In [33]: sqrtdenest(e)
Out[33]: -2 + √5
```
https://docs.sympy.org/latest/modules/simplify/simplify.html#sympy.simplify.sqrtdenest.sqrtdenest
